pydeseq2/ds.py

```diff
@@ -356,9 +383,12 @@ def lfc_shrink(self, coeff: str) -> None:
             self.results_df["log2FoldChange"] = self.LFC.iloc[:, coeff_idx] / np.log(2)
             self.results_df["lfcSE"] = self.SE / np.log(2)
 
+            # Get the corrresponding factor, tested and reference levels of the shrunk
+            # coefficient
+            split_coeff = coeff.split("_")
             print(
                 f"Shrunk Log2 fold change & Wald test p-value: "
-                f"{self.contrast[0]} {self.contrast[1]} vs {self.contrast[2]}"
+                f"{split_coeff[0]} {split_coeff[1]} vs {split_coeff[3]}"
```

Comment:
I got a bit lost, I was expecting `split_coeff[2]` instead of 3. can you write a short comment so it's easier to follow on what's in the `coeff` (or `split_coeff`) when reading the code?